charts/dynatrace-oneagent-operator/v0.7.0/Chart.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 Dynatrace LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v2
+name: dynatrace-oneagent-operator
+description: The Dynatrace OneAgent Operator Helm chart for Kubernetes and Openshift
+icon: https://assets.dynatrace.com/global/resources/Signet_Logo_RGB_CP_512x512px.png
```

Comment:
can u please help to upload the icon to the parent directly and reference it as `file://../Signet_Logo_RGB_CP_512x512px.png`, therefore air-gapped env user can view it properly, thanks

Reply:
Sure - done!